src/Themes/ui/ThemeEditorModal.tsx

```diff
@@ -81,15 +82,22 @@ export function ThemeEditorModal(props: IProps): React.ReactElement {
 
   function onThemeChange(event: React.ChangeEvent<HTMLInputElement>): void {
     try {
-      const importedTheme = JSON.parse(event.target.value);
-      if (typeof importedTheme !== "object") return;
+      const importedTheme = JSON.parse(event.target.value) as typeof Settings.theme;
+      if (importedTheme == null) {
+        throw new Error("Theme data must not be null or undefined.");
+      }
+      if (typeof importedTheme !== "object") {
+        throw new Error(`Theme data is invalid.`);
+      }
       setCustomTheme(importedTheme);
       for (const key of Object.keys(importedTheme)) {
         Settings.theme[key] = importedTheme[key];
       }
       ThemeEvents.emit();
-    } catch (err) {
-      // ignore
+    } catch (error) {
+      console.error(`Theme data is invalid. Data: ${event.target.value}.`);
+      console.error(error);
+      dialogBoxCreate(`Invalid theme. ${String(error)}`);
```

Comment:
Per disucussion on the other PR, possibly still just:
```suggestion
      dialogBoxCreate(`Invalid theme. ${error}`);
```